### DISCUSSION TITLE
How can I disable escaping the tour by pressing ESC key?

### DISCUSSION THREAD

**Discussion by marinaadlc** [gilbarbara/react-joyride#795] [Q&A] [answered]
Hi!
I am using the controlled mode and have disabled the escape button, but when pressing the ESC key the tour can be closed. 
Can this option be disabled?

Thank you!

**Comment by gilbarbara** [comment, marked as answer]
@marinaadlc 
Not sure what you meant with the "escape button". Is it the close button?
Are you setting the `disableCloseOnEsc` prop? Did you check the [docs](https://docs.react-joyride.com/props)?
